### ISSUE TITLE
[Project Addition] : Chest X-ray - Tuberculosis Disease Prediction

### ISSUE THREAD

**Issue by aaradhyasinghgaur** [abhisheks008/DL-Simplified#755]
### Deep Learning Simplified Repository (Proposing new issue)
:red_circle: **Project Title** : Chest X-ray - 13 Diseases Prediction</br>
:red_circle: **Aim** : To predict and classify between Normal x-ray and tuberculosis diseases using chest X-ray images, a convolutional neural network (CNN) can be employed. This deep learning model will process the X-ray images to identify characteristic features and patterns indicative of each disease. The trained model will output a probability score for each disease, allowing for accurate and efficient diagnosis based on the visual data contained in the chest X-rays.</br>
:red_circle: **Dataset** : https://www.kaggle.com/datasets/tawsifurrahman/tuberculosis-tb-chest-xray-dataset </br>
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : Aaradhya Singh 
- GitHub Profile Link : https://github.com/kyra-09
- Email ID : aaradhyasinghgaur@gmail.com
- Participant ID (if applicable):
- Approach for this Project :
   1. Data Preparation: Collect and preprocess a balanced dataset of real and fake face images, including normalization, resizing, and augmentation.
  2.  Base Model Selection:  EfficientNetB0,VGG16 , Xception , InceptionV3 like 5 different models  excluding its top layers, to leverage its learned features.
  3.  Model Construction: Add custom layers on top of the base model for binary classification, compiling with appropriate loss and metrics.
  4.  Initial Training: Train the model with the base layers frozen to only update the new layers.
  5.  Fine-Tuning: Unfreeze some or all of the base model layers and continue training with a lower learning rate to fine-tune the entire network.
  6.) EDA analysis.
  7.) Comaprioson using performance matrices such as accuracy scores , confusion matrix etc.
- What is your participant role? (Mention the Open Source program) - GSSOC-2024 Contributor

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by abhisheks008** [comment]
Assigned @aaradhyasinghgaur
